README.rst

```diff
@@ -2,56 +2,129 @@
 Nurse Scheduling
 ================
 
-THIS IS A WORK IN PROGRESS! ONLY TEMPLATE FILES HERE AT THE MOMENT.
+This is a demo of a nurse scheduling model developed by Ikeda, Nakamura
+and Humble (INH). 
 
-Short blurb on what this demo does.
+The nurse scheduling problem seeks to find an optimal assignment
+for a group of nurses, under constraints of scheduling and personnel.
+INH developed a model which is a simplified representation of a real-world 
+nursing facility.
 
-If your demo produces a visualization, consider displaying an image of it here.
+In the general nurse scheduling problem, there are three types of constraints,
+which are mentioned here to provide background for INH's constraints.
+These types of constraints, in the general problem, are:
 
-.. image:: dwave_logo.png
+1) Both upper and lower limits on the number of breaks.
+2) The number of nurses on duty for each shift slot.
+3) For each individual nurse, upper and lower limits on the time interval 
+   between days of duty.
 
+These three types of constraints combine to ensure sufficient nurses
+on duty at all times, without overworking any particular nurse.
+
+INH took these general constraints and formulated them as a QUBO.
```

Comment:
I thought I had commented on this but maybe I accidentally cancelled the comment: "INH took these general constraints and formulated them as a QUBO" says that the QUBO is based on the general constraints "which are mentioned here to provide background for INH's constraints".